### ISSUE TITLE
yolo11.yaml files

### ISSUE THREAD

**Issue by apiszcz** [ultralytics/ultralytics#22820]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

In an attempt to create a yolo model with no training the recommendation is to load the yolo??.yaml file and save the weights to a .pt file. Reviewing the source cfg/models/yolo/11 contains a single yolo11n.yaml model which appears to be for the 'n' architecture.  Where are the yaml configuration architecture files for the sizes located? 's,m,l,x' for yolov11?

./ultralytics/cfg/datasets/SKU-110K.yaml
./ultralytics/cfg/models/11/yolo11-cls-resnet18.yaml
./ultralytics/cfg/models/11/yolo11-cls.yaml
./ultralytics/cfg/models/11/yolo11-obb.yaml
./ultralytics/cfg/models/11/yolo11-pose.yaml
./ultralytics/cfg/models/11/yolo11-seg.yaml
./ultralytics/cfg/models/11/yolo11.yaml
./ultralytics/cfg/models/11/yoloe-11-seg.yaml
./ultralytics/cfg/models/11/yoloe-11.yaml

the yolo10 architectures are avaialble
./ultralytics/cfg/models/v10/yolov10b.yaml
./ultralytics/cfg/models/v10/yolov10l.yaml
./ultralytics/cfg/models/v10/yolov10m.yaml
./ultralytics/cfg/models/v10/yolov10n.yaml
./ultralytics/cfg/models/v10/yolov10s.yaml
./ultralytics/cfg/models/v10/yolov10x.yaml

thank you

### Additional

This post below  indicates the files are in the cfg/models folder. 
However only the 'n' parameters are present in the cfg/models file. 

[glenn-jocher](https://github.com/glenn-jocher)
[on May 28](https://github.com/orgs/ultralytics/discussions/20847#discussioncomment-13297860)
Maintainer
Hi @Lingpheng! The YOLO11 model variants (n, s, m, l, x) are defined by their depth and width multipliers within the main yolo11.yaml configuration file. You don't need separate YAML files for each variant.

To train from scratch without pretrained weights, you can use:

from ultralytics import YOLO

# Load architecture only (no pretrained weights)
model = YOLO("yolo11n.yaml")  # or yolo11s.yaml, yolo11m.yaml, etc.
results = model.train(data="datas.yaml", epochs=50, patience=15)
The model variant YAML files (yolo11n.yaml, yolo11m.yaml, etc.) are located in the ultralytics/cfg/models/11/ directory and define the specific architecture parameters for each model size.

Answer selected by [Y-T-G](https://github.com/Y-T-G)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @apiszcz, thank you for opening this issue and for your interest in Ultralytics YOLO11 🚀  
This is an automated triage response — an Ultralytics engineer will review your question and follow up with you here soon 🙂

From your description, it looks like you’re asking about the YOLO11 architecture YAML definitions for different model sizes (`n, s, m, l, x`) and how they relate to the single `yolo11.yaml` file you’re seeing in `cfg/models/11`. An Ultralytics engineer will clarify the current layout and recommended workflow for creating models (including architecture-only, no-training setups) in the latest version of the repository.

In the meantime, you may find it helpful to review the official docs and examples, as many common questions on configuration and training from scratch are covered there:

- Visit the [Docs](https://docs.ultralytics.com/) 📚  
- See [Python examples](https://docs.ultralytics.com/usage/python/)  
- See [CLI examples](https://docs.ultralytics.com/usage/cli/)

If this is a 🐛 Bug Report (for example, if YAML files referenced in the docs or discussions are missing, paths are incorrect, or behavior does not match expectations), please also provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) (MRE) so the team can investigate efficiently. This can include:
- Exact code you ran
- Full error message or stack trace (if any)
- Your environment details (OS, Python version, `ultralytics` version, how installed)

If this is a custom training ❓ Question (such as how to configure or instantiate different YOLO11 sizes, or how to save architecture-only models to `.pt` without training), please provide:
- The exact command or Python snippet you’re using
- Any config/YAML snippets relevant to your workflow
- A brief description of your goal (e.g., “create YOLO11-s architecture only, no training, save to .pt”)
- Confirmation that you’ve reviewed our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) if training is involved

You’re also very welcome to connect with the community for more interactive help and discussion:
- Real-time chat on [Discord](https://discord.com/invite/ultralytics) 🎧  
- In-depth Q&A on [Discourse](https://community.ultralytics.com/)  
- Community threads on [Subreddit](https://www.reddit.com/r/Ultralytics/)

## Upgrade

Before the team digs in, please make sure you’re on the latest `ultralytics` package (and dependencies), as file layouts and defaults can evolve between versions. In a clean [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/), upgrade with:

```bash
pip install -U ultralytics
```

Also ensure all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are installed so your environment matches the current repository expectations.

## Environments

YOLO can be run in any of the following up-to-date verified environments (with all dependencies such as [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU:  
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a>  
  <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>  
  <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing ✅  
CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by apiszcz** [comment]
Disregard, I see YOLO 'auto' scales for the different sizes.

**Comment by Y-T-G** [comment]
The same base file is reused for all the scales depending on the suffix used when loading:

https://docs.ultralytics.com/guides/model-yaml-config#parameters-section
